Code/Editor/MainStatusBar.cpp

```diff
@@ -342,7 +357,19 @@ void SourceControlItem::InitMenu()
 
         connect(m_settingsAction, &QAction::triggered, this, [&]()
         {
-            GetIEditor()->GetSourceControl()->ShowSettings();
+            // LEGACY note - GetIEditor and GetSourceControl are both legacy functions
+            // but are currently the only way to actually show the source control settings.
+            // They come from an editor plugin which should be moved to a gem eventually instead.
```

Comment:
We should create a GHI to capture this (if there isn't one already)

Reply:
Will do.

Reply:
Its actually really tempting too since there are only 4 remaining plugins and besides the Component Property Editor (which maybe is replaced by DPE?) the others are just a handful of lines of code.  Eliminateing them  would allow us to remove the entire legacy plugin system.

Reply:
https://github.com/o3de/o3de/issues/17571